yuuna/lib/src/pages/implementations/player_source_page.dart

```diff
@@ -1207,6 +1225,7 @@ class _PlayerSourcePageState extends BaseSourcePageState<PlayerSourcePage>
             child: Row(
               children: [
                 const Space.small(),
+                buildPinButton(),
```

Comment:
I don't think this is an intuitive position to place this menu item. This needs to go elsewhere.